src/modules/fetchComment.js

```diff
@@ -19,7 +20,8 @@ const displayComments = (data) => {
       </tr>`;
     });
     table.innerHTML = tableHTML;
-    commentSpan.textContent = `${data.length} comments for this movie`;
+    // commentSpan.textContent = `${data.length} comments for this movie`;
```

Comment:
- [ ] Kindly check and remove all commented code from your fetchComment.js file. 🧹 

It is best practice to remove any commented-out code that is no longer used to avoid cluttering the codebase and making it harder to read.